app/models/concerns/team_statistics.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+# Match statistics for team
+module TeamStatistics
+  extend ActiveSupport::Concern
+
+  def matches(section = nil)
+    home_matches(section) + away_matches(section)
```

Comment:
`home_matches` and `away_matches` must be implemented by including model.  Can it be added in the module and overridden by the model?